### ISSUE TITLE
Temporal smoothing 

### ISSUE THREAD

**Issue by tsing90** [CUHKSZ-TQL/EverybodyDanceNow_reproduce_pytorch#12]
Hi, 

It's a great work done on reproducing the paper. After reading your code, it seems that temporal smoothing has not been implemented. Am I right? thanks

**Comment by yanx27** [comment]
Yes, the code does not contain temporal smoothing, so if the action is too fast, the background will not be stable enough, and we will continue to improve it in the future, thank u for you feedback :)

**Comment by tsing90** [comment]
thanks for your reply, I'm looking forward to that implementation which I think is quite important for video quality.
